tools/scripts/build/build-mobile-curriculum.js

```diff
@@ -33,8 +48,34 @@ exports.buildMobileCurriculum = function buildMobileCurriculum(json) {
     }
   }
 
-  function writeToFile(filename, json) {
-    const fullPath = `${mobileStaticPath}/mobile/${filename}.json`;
+  function getBlockDescription(superBlockKey, blockKey) {
+    try {
+      const intros = JSON.parse(fs.readFileSync(blockIntroPath));
+
+      const introValues = intros[superBlockKey]['blocks'][blockKey]['intro'];
+
+      return introValues.join('');
+    } catch (e) {
+      return '';
+    }
+  }
+
+  function getSuperBlockNames() {
+    try {
+      const superBlocks = JSON.parse(fs.readFileSync(blockIntroPath));
+
+      return Object.keys(superBlocks).map(
+        superBlock => superBlocks[superBlock].title
+      );
+    } catch (e) {
+      return [];
+    }
+  }
+
+  function writeToFile(fileName, json) {
+    fileName = fileName.replace(/\//, '-');
```

Comment:
```suggestion
```
This replace appears twice and it's a bit weird that `writeToFile(filename, json)` doesn't simply write `json` to a file called `filename`, how about pulling this out into dedicated function?